libs/messenger-bundle/src/DependencyInjection/KeboolaMessengerExtension.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\MessengerBundle\DependencyInjection;
+
+use InvalidArgumentException;
+use Symfony\Component\Config\Definition\Configurator\DefinitionConfigurator;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\AbstractExtension;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use Symfony\Component\DependencyInjection\Loader\PhpFileLoader;
+
+class KeboolaMessengerExtension extends AbstractExtension
```

Comment:
Neni pokryte unit testama.

Reply:
No tohle mi prislo enough pokryty tim, ze prochazi funkcni testy (`AwsSqsConsumptionTest` apod.). Pokud by tu byla chyba a blbe to nakonfigurovalo appku, funkcni test by neprosel.

Reply:
Jeste jsem nad tim premyslel a par veci by tu mozna stalo za to testnout explicitne. Dodelam to zase v ramci toho #169

Reply:
To jo, nakonec mi to dalo docela zabrat :sweat_smile: ale podarilo se mi neco vyplodit. Hodil jsem to do #169.